x-pack/solutions/security/plugins/security_solution/public/flyout/shared/components/flyout_history_row.tsx

```diff
@@ -160,20 +174,54 @@ interface GenericHistoryRowProps extends FlyoutHistoryRowProps {
  */
 export const GenericHistoryRow: FC<GenericHistoryRowProps> = memo(
   ({ item, index, title, icon, name, dataTestSubj }) => {
+    const { euiTheme } = useEuiTheme();
     const { openFlyout } = useExpandableFlyoutApi();
     const onClick = useCallback(() => {
-      openFlyout({ right: item });
-    }, [openFlyout, item]);
+      openFlyout({ right: item.panel });
+    }, [openFlyout, item.panel]);
 
     return (
       <EuiContextMenuItem
         key={index}
         onClick={onClick}
         icon={icon}
+        css={css`
+          align-items: flex-start;
+          padding: ${euiTheme.size.s} ${euiTheme.size.m};
+        `}
         data-test-subj={`${index}-${dataTestSubj ?? GENERIC_HISTORY_ROW_TEST_ID}`}
       >
-        <i>{`${name}: `}</i>
-        {title}
+        <EuiFlexGroup direction="column" gutterSize="xs">
+          <EuiFlexItem
+            css={css`
+              flex-direction: row;
+            `}
+          >
+            <span
```

Comment:
I would take these 2 `span` out into a `formattedTitle` variable, but just a personal preference.

Style component is not that bad after all huh?

Reply:
yeah good idea again here, I'll build a component for this, will be much cleaner, thanks!